### PR TITLE
docs: add basic dashboard docs

### PR DESCRIPTION
**Pull request by antbern**
## Motivation and Context

Describe the usage of the dashboard 💯 

## Checklist

Please ensure that your PR meets the following requirements:

### General

- [ ] Code follows project style and guidelines.
- [ ] No redundant or duplicate code.
- [ ] No added new dependencies without clear justification and approval.

### Documentation

- [ ] Added/updated relevant documentation (e.g., README, inline comments).
- [ ] Added a CHANGELOG entry for major changes.

### Testing

- [ ] Added relevant test cases, leveraging tools like `rstest` or similar for reusable tests.
- [ ] Tests pass locally.

### Build & CI/CD

- [ ] Confirmed no CI/CD pipeline issues.
- [ ] Updated build scripts where necessary.

## Comments to Reviewer

Please provide any comments or points of consideration for the reviewers, such as specific areas of focus, potential edge cases, or context that would aid the review process.

### Reviewer Responsibilities

- [ ] Code adheres to style and guidelines.
- [ ] All tests are appropriately covered and they pass.
- [ ] Consider performance and/or security impacts.
- [ ] Constructive feedback is provided.

### CODE REVIEW

File: docs/docs/Dashboard/Dashboard.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+To retrieve the configuration needed for the SDK, go to the _Project Details_ page of your project.
+Under _Setup Guide_ you will find a template configuration JSON that you can use to initialize the SDK.
+See also [Configuration](../SDK%20Configuration/Configuration.md) for details on configuring the SDK.
+
+## Configuring Secrets
+
+To use any of the supported [KYC providers](../SDK%20Modules/KYC%20Onboarding.md) for enabling Know Your Customer (KYC) on your instance, visit the _Secrets_ tab under _Project Settings_.
+On this page, you can enter your access credentials for respective service. Note that you need to do this for the `live` and `test` instace separately.
+
+???+ info
```
Comment (mighty840) [inline docs/docs/Dashboard/Dashboard.md:65]: Is i then possible to override the secret values by uploading new ones?

Comment (antbern) [reply docs/docs/Dashboard/Dashboard.md:66]: Yes you can save new secrets at any time. Updated to clarify!

File: docs/docs/Dashboard/Dashboard.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+1. Login using GitHub.
+2. Manage your active projects.
+3. Retrieve the [configuration](../SDK%20Configuration/Configuration.md) needed to use the ETOPay SDK.
+3. Configure secrets required for integrating with [KYC providers](../SDK%20Modules/KYC%20Onboarding.md).
+
+## Login using GitHub
+
+Open the [dashboard](https://etopayapp.etospheres.com) and press the button to login through your [GitHub](https://github.com) account.
+By logging in an using our platform you agree to the [Terms of Service](https://etopayapp.etospheres.com/terms_of_service) and [Privacy Policy](https://etopayapp.etospheres.com/privacy_policy).
```
Comment (kristifidani) [inline docs/docs/Dashboard/Dashboard.md:13]:
```suggestion
By logging in and using our platform you agree to the [Terms of Service](https://etopayapp.etospheres.com/terms_of_service) and [Privacy Policy](https://etopayapp.etospheres.com/privacy_policy).
```

File: docs/docs/Dashboard/Dashboard.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+## Manage Projects
+
+
+### Creating a Project
+
+After logging in for the first time, you'll be greated with a page that allows you to create a new project by signing up for your preferred plan.
+
+???+ info
+    The payment is entirely handled through [Stripe](https://stripe.com). We do not store or have access you your payment details at any time.
```
Comment (kristifidani) [inline docs/docs/Dashboard/Dashboard.md:26]:
```suggestion
    The payment is entirely handled through [Stripe](https://stripe.com). We do not store or have access to your payment details at any time.
```

File: docs/docs/Dashboard/Dashboard.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    After paying and creating a project you can manage your subscription at any time through the _Manage Subscription_ navigation link.
+
+When signing up for a new plan a new project is created in your list of projects. You can create a new project, with your preferred plan, using the _Create New Project_ button.
```
Comment (kristifidani) [inline docs/docs/Dashboard/Dashboard.md:29]:
```suggestion
When signing up for a new plan, a new project is created add added to your list of projects. You can create a new project, with your preferred plan, using the _Create New Project_ button.
```

File: docs/docs/Dashboard/Dashboard.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+Each project includes two instances of the ETOPay services:
+
+1. A `test` instance used for testing during integration and development.
+2. A `live` instance used for the actual production usage.
+
+After creating a project, follow the _Setup Guide_ shown on the _Project Details_ page to perform the required one-time setup of the Keycloak administrator accounts that you will use to manage your end-users.
```
Comment (kristifidani) [inline docs/docs/Dashboard/Dashboard.md:36]:
```suggestion
After creating a project, follow the _Setup Guide_ shown on the _Project Details_ page to perform the required one-time setup of the Keycloak administrator account that you will use to manage your end-users.
```

File: docs/docs/Dashboard/Dashboard.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+## Retrieving Configuration
+
+To retrieve the configuration needed for the SDK, go to the _Project Details_ page of your project.
+Under _Setup Guide_ you will find a template configuration JSON that you can use to initialize the SDK.
+See also [Configuration](../SDK%20Configuration/Configuration.md) for details on configuring the SDK.
+
+## Configuring Secrets
+
+To use any of the supported [KYC providers](../SDK%20Modules/KYC%20Onboarding.md) for enabling Know Your Customer (KYC) on your instance, visit the _Secrets_ tab under _Project Settings_.
+On this page, you can enter your access credentials for respective provider. Note that you need to do this for the `live` and `test` instace separately.
```
Comment (kristifidani) [inline docs/docs/Dashboard/Dashboard.md:63]: Do we also need to mention here that currently we support Postident and Viviswap?

```suggestion
On this page, you can enter your access credentials for the respective provider. Note that you need to do this for the `live` and `test` instance separately.
```

**Review by kristifidani — COMMENTED** [review]
Nice to have some docs for the dashboard 💯  

I added some comments, mostly small typos here and there which I noticed while reading

File: docs/docs/Dashboard/Dashboard.md (hunk 6 above)
Comment (antbern) [reply docs/docs/Dashboard/Dashboard.md]: In my opinion not since we link to the page about KYC. Better not to duplicate too much information IMO ...

Comment (kristifidani) [reply docs/docs/Dashboard/Dashboard.md]: Alright 👍

**Review by kristifidani — APPROVED** [review]
